addons/website/static/src/js/utils.js

```diff
@@ -4,7 +4,23 @@ import { intersection } from "@web/core/utils/arrays";
 import { csrf_token, _t } from "@web/legacy/js/services/core";
 import { renderToElement } from "@web/core/utils/render";
 
-
+let _languagesPerWebsiteCache;
+/**
+ * Allows to retrieve the list of available language for the website with a cache of 10 seconds
+ *
+ * @param {Function} rpc
+ * @returns {Array} list of languages
+ */
+async function loadLanguages(rpc) {
+    const currentTimestamp = Math.round(+new Date() / 1000);
+    if (!_languagesPerWebsiteCache || currentTimestamp - _languagesPerWebsiteCache.timestamp > 10) {
+        _languagesPerWebsiteCache = {
+            languages: rpc({ route: "/website/get_languages" }),
+            timestamp: Math.round(+new Date() / 1000),
```

Comment:
Maybe do something like `m2oRpcCache` to have a cache for the whole edit mode duration only.